tonic_validate/metrics/response_length_metric.py

```diff
@@ -9,16 +9,15 @@
 
 
 class ResponseLengthMetric(BinaryMetric):
-    """Checks that response is within a certain number of characters."""
-
     def __init__(
         self,
         name: str,
         min_length: Optional[int] = None,
         max_length: Optional[int] = None,
     ):
         """
-        Create a metric that checks if the length of the LLM response is within a given length.
+        Create a binary metric that checks if the length of the LLM response is within a given length.
```

Comment:
Creates a binary metric that checks whether the LLM response length falls within a given range.